connectivity/FEATURE_BLE/source/cordio/TESTS/cordio_hci/driver/CMakeLists.txt

```diff
@@ -3,10 +3,10 @@
 
 cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
 
-set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../.. CACHE INTERNAL "")
+set(mbed-os_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../.. CACHE INTERNAL "")
```

Comment:
As a current implementation, every greentea test has its CMakeLists.txt similar to application CMake. Why do we need to change into `mbed-os_SOURCE_DIR`?

Reply:
I see, this is setting it. I missed it during refactoring, I'll remove all